sharepoint/sharepoint-ps/sharepoint-online/Start-SPOCopilotAgentInsightsReport.md

```diff
@@ -10,13 +10,16 @@ ms.author: bhagatshweta
 ms.reviewer:
 manager: hikakar
 ---
- 
-# Start-SPOCopilotAgentInsightsReport
+
+ # Start-SPOCopilotAgentInsightsReport
 
 ## SYNOPSIS
 
 Using this cmdlet, administrators may trigger the build of a new Copilot agent insight report for the specified number of days.
 
+> [!NOTE]
+> The cmdlets are currently in development and likely to be rolled out for private preview in early March 2025 and will be subsequently expanded to global audience/users. 
```

Comment:
>The cmdlets are currently in development and likely to be rolled out for private preview in early March 2025 and will be subsequently expanded to global audience/users.  [](http://example.com/codeflow?start=2&length=169)

could we get a more polished string? e.g. avoid using "likely". I'd also avoid putting dates here, since these could change.

e.g., I see some cmdlets use the simple "This cmdlet is currently under private preview." which is simple, clear and sends the right message.

@AdwoodMicrosoft could suggest on the right message to use.

Reply:
applies to the other PR as well.